.unbroken_exclusions

```diff
@@ -13,6 +14,8 @@
 !packages/scripts/node_modules
 !packages/@react-native-windows/find-repo-root/node_modules
 !packages/@rnw-scripts/create-github-releases/node_modules
+!packages/@rnw-scripts/doxysaurus/node_modules
```

Comment:
Mentioned offline, but I think this would be better in the root packages directory instead of `@rnw-scripts`. Most everything should still work if you move + rename in package.json, but you might need to update some relative paths. E.g. the launch.json in .vscode.

Reply:
Let me do it later. It seems to be a big change at this point.